mteb/evaluation/evaluators/SummarizationEvaluator.py

```diff
@@ -62,6 +61,10 @@ def __call__(self, model):
                 dot_pred_scores.append(dot_max_score)
                 human_scores.append(human_eval_score)
 
+            if (len(set(human_scores)) == 1) or (len(set(dot_pred_scores)) == 1) or (len(set(cosine_pred_scores)) == 1):
+                logging.info(f"Skipping sample {i} due to equal scores")
```

Comment:
I think it's better to use the local logger instead of the global one 
```suggestion
                logger.info(f"Skipping sample {i} due to equal scores")
```